core/transport/src/main/proto/grpc/block_master.proto

```diff
@@ -246,6 +257,20 @@ message GetRegisterLeasePResponse {
   optional GetRegisterLeasePOptions options = 4;
 }
 
+message WorkerPreRegisterInfo {
+  optional PreRegisterCommandType preRegisterCommandType = 1;
+  optional string clusterId = 2;
+  optional int64 workerId = 3;
+}
+
+enum PreRegisterCommandType {
+  UNKNOWN = 0;
```

Comment:
what is UNKNOWN for?

Reply:
`UNKNOWN` is a default value. `UNKNOWN` should not appear unless an error occurs
I refer to the heartbeat CommandType
core/transport/src/main/proto/grpc/common.proto
```
enum CommandType {
  Unknown = 0;
  Nothing = 1;
  Register = 2; // Ask the worker to re-register.
 //...
}
```